src/types.ts

```diff
@@ -39,6 +39,8 @@ export interface Column<TRow, TSummaryRow = unknown> {
   readonly colSpan?: Maybe<(args: ColSpanArgs<TRow, TSummaryRow>) => Maybe<number>>;
   /** Determines whether column is frozen or not */
   readonly frozen?: Maybe<boolean>;
+  /** Determines whether column is right frozen or not */
+  readonly rightFrozen?: Maybe<boolean>;
```

Comment:
I would rather not add a new prop that can cause conflicts. What happens when both `rightFrozen` and `frozen` props are set? We can instead change `frozen` to `boolean | 'left' | 'right'`.